src/CustomerProfile.JS

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from "react";
+
+const CustomerProfile = (props) => {
+  console.log(props);
+  const [profileId, setProfileId] = useState("");
+
+  useEffect(() => {
+    setProfileId(props.id);
+  }, [props.id]);
+
+  console.log(props.id);
```

Comment:
You don't need an effect at this point in the exercise, because we're just displaying `props.id` rather than fetching data from somewhere else. But it's fine to have this here as a placeholder.

`useEffect` will make more sense in [Lesson 3 point 21](https://github.com/CodeYourFuture/cyf-hotel-react#21-display-a-customer-profile---step-2)